src/app/not-found.tsx

```diff
@@ -1,5 +1,9 @@
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <p className="text-acm-navy flex h-screen w-screen items-center justify-center text-[60px] font-bold">
```

Comment:
```suggestion
    <p className="text-acm-navy flex h-screen w-screen items-center justify-center text-[60px] font-bold">
```
the figma says 60px, but instead of text-[60px] use one of the predefined tailwind font size classes that best match the figma

https://tailwindcss.com/docs/font-size